curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3485f0b20591d26d2a1.md

```diff
@@ -7,14 +7,14 @@ dashedName: step-19
 
 # --description--
 
-Smooth out your images a bit by giving the `#gallery img` selector a `border-radius` property of `10px`.
+Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property with `10px` set as the value.
```

Comment:
```suggestion
Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property of `10px`.
```

Reply:
End of sentence code block.

Reply:
I'm fine with your original change here.